### PR TITLE
[Merged by Bors] - hare: only check identity active when synced

### PR DESCRIPTION
**Pull request by countvonzero**

### CODE REVIEW

**Review by dshulyak — APPROVED** [review]

File: hare/eligibility/oracle.go [hunk 1]
```diff
@@ -516,7 +516,6 @@ func (o *Oracle) IsIdentityActiveOnConsensusView(ctx context.Context, edID types
 	}()
 	actives, err := o.actives(ctx, layer)
 	if err != nil {
-		o.WithContext(ctx).With().Error("error getting active set", layer, log.Err(err))
```
Comment (piersy) [inline hare/eligibility/oracle.go:519]: Why is this log removed?

Comment (countvonzero) [reply hare/eligibility/oracle.go:519]: because the returned error is already logged at call sites.

**Comment by countvonzero** [comment]
bors merge

**Comment by countvonzero** [comment]
bors merge

**Pull request closed without merging**